src/tokens/saga.ts

```diff
@@ -273,7 +323,12 @@ export function* watchAccountFundedOrLiquidated() {
   }
 }
 
+export function* watchFetchTokenPriceHistory() {
+  yield* takeEvery(fetchPriceHistoryStart.type, safely(fetchTokenPriceHistorySaga))
```

Comment:
What do you think about `takeLatest` instead? If someone switches to a different timescale for instance before the data has loaded i think we should cancel the old request and start the new one right away.

Reply:
Was thinking we'd probably fetch more data than needed then filter down in the frontend to avoid a fetching and for each timespan change, but that's a bit outside of the scope of this PR and using `takeLatest` makes sense in either case.

Reply:
Fixed in d0265a568d2bd337a1ec85acfca25b2cd39db72b!